org.eclipse.richbeans.widgets/src/org/eclipse/richbeans/widgets/shuffle/ShuffleConfiguration.java

```diff
@@ -0,0 +1,126 @@
+package org.eclipse.richbeans.widgets.shuffle;
+
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * 
+ * Class for configuring a ShuffleWidget
+ * 
+ * @author Matthew Gerring
+ *
+ */
+public class ShuffleConfiguration {
+
+    private String fromToolipText, toToolipText;    
+	private List<Object> fromList, toList;
+	private boolean fromReorder=false, toReorder=true;
+	private String fromLabel, toLabel;
+
+	public ShuffleConfiguration() {
+		fromList = Collections.emptyList();
+		toList   = Collections.emptyList();
+	}
+    
+	public String getFromToolipText() {
+		return fromToolipText;
+	}
+	public void setFromToolipText(String fromToolipText) {
+		String old = this.fromToolipText;
+		this.fromToolipText = fromToolipText;
+		firePropertyChange("fromToolipText", old, fromToolipText);
+	}
+	public String getToToolipText() {
+		return toToolipText;
+	}
+	public void setToToolipText(String toToolipText) {
+		String old = this.toToolipText;
+		this.toToolipText = toToolipText;
+		firePropertyChange("toToolipText", old, toToolipText);
+    }
+	
+    public List<Object> getFromList() {
+		return fromList;
+	}
+	public void setFromList(List<Object> fromList) {
+		List<Object> old = this.fromList;
+		this.fromList = fromList;
+		firePropertyChange("fromList", old, fromList);
+	}
+	public List<Object> getToList() {
+		return toList;
+	}
+	public void setToList(List<Object> toList) {
+		List<Object> old = this.toList;
+		this.toList = toList;
+		firePropertyChange("toList", old, toList);
+	}
+	
+	public boolean isFromReorder() {
+		return fromReorder;
+	}
+
+	public void setFromReorder(boolean fromReorder) {
```

Comment:
This marks the from box (e.g. experiments from the database?) to be re-orderable. We don't need this in the current usecase. But I can see it could be useful.